Sources/Rendering/Core/CellPicker/index.js

```diff
@@ -1,15 +1,25 @@
 import macro from 'vtk.js/Sources/macros';
-import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import vtkCellTypes from 'vtk.js/Sources/Common/DataModel/CellTypes';
+import vtkLine from 'vtk.js/Sources/Common/DataModel/Line';
 import vtkPicker from 'vtk.js/Sources/Rendering/Core/Picker';
-import vtkPoints from 'vtk.js/Sources/Common/Core/Points';
+import vtkPolyLine from 'vtk.js/Sources/Common/DataModel/PolyLine';
 import vtkTriangle from 'vtk.js/Sources/Common/DataModel/Triangle';
-
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import { CellType } from 'vtk.js/Sources/Common/DataModel/CellTypes/Constants';
 import { vec3 } from 'gl-matrix';
 
 // ----------------------------------------------------------------------------
 // Global methods
 // ----------------------------------------------------------------------------
 
+function getCellTempMap() {
+  return {
+    [CellType.VTK_LINE]: vtkLine.newInstance(),
+    [CellType.VTK_POLY_LINE]: vtkPolyLine.newInstance(),
+    [CellType.VTK_TRIANGLE]: vtkTriangle.newInstance(),
+  };
```

Comment:
`return new Map(CellType.keys().map((cellType) => return [cellType, CellType.getClassNameFromTypeId(cellType)])`

Reply:
imo this is elegant, but we can't use it.
We don't support all cells, and we would need to dynamically import the relevant module from the class string, and I don't know how to do this properly without creating problems.
If someday we supports all the cells listed in `vtk-js/Sources/Common/DataModel/CellTypes/Constants.js` and also have a map directly with the corresponding modules we shall use it.

Reply:
fair enough